pkg/controller/stats/aggregator.go

```diff
@@ -245,6 +253,32 @@ func (a *Aggregator) ListAntreaClusterNetworkPolicyStats() []statsv1alpha1.Antre
 	return stats
 }
 
+func (a *Aggregator) ListMulticastGroups() []statsv1alpha1.MulticastGroup {
+	stats := make([]statsv1alpha1.MulticastGroup, 0, len(a.groupNodePodsMap))
+	for group, nodePods := range a.groupNodePodsMap {
+		listPods := make([]statsv1alpha1.PodReference, 0)
```

Comment:
```suggestion
		allPods := make([]statsv1alpha1.PodReference, 0)
```
listPods sounds like a function/action

Reply:
updated